docs/Measuring_Resonances.md

```diff
@@ -102,6 +102,38 @@ accel_chip: adxl345
 probe_points:
     100,100,20  # an example
 ```
+
+### Primary MCU
+Add the following to the printer.cfg file:
+```
+[adxl345]
+cs_pin: AB0 # The MCU pin connected to the CS pin on your sensor
```

Comment:
In my experience, it is frequently necessary to also specify spi_bus. Unfortunately, what's currently lacking in Klipper documentation is a list of the possible values for this parameter for the supported MCUs. So I had to dig up the names from the source code directly. Maybe there is something available? Even if not, it would probably make sense to mention this requirement to the users and direct them in the right direction how to figure out the appropriate values.

Reply:
I'll add a reference to the SPI section of the config document. I think this line settles it: 
```
#   If the micro-controller supports multiple SPI busses then one may
#   specify the micro-controller bus name here. The default depends on
#   the type of micro-controller.
```

Reply:
If it's worth it, I think the names of available spi buses should be included in config files for boards

Reply:
Yes, maybe having that in the board config files would be the best. In fact, those configs could have examples of configs for the connected ADXL345, similar to having display sections. Then maybe the folks who successfully connected the accelerometers to their MCUs could share their findings there, e.g. which SPI buses are appropriate for connection and under which conditions, and whether the 5V-tolerant accelerometer required for a given board.